lookout/style/format/analyzer.py

```diff
@@ -275,12 +275,16 @@ def train(cls, ptr: ReferencePointer, config: Mapping[str, Any], data_service: D
                 _log.warning("skipped %d %s files: too few samples (%d/%d)",
                              len(files), language, X_train.shape[0], lower_bound_instances)
                 continue
-            _log.debug("training the rules model")
+            _log.info("extracted %d samples to train, searching for the best hyperparameters",
+                      X_train.shape[0])
             optimizer = Optimizer(**lang_config["optimizer"], random_state=random_state)
             best_score, best_params = optimizer.optimize(X_train, y_train)
-            _log.debug("score of the best estimator found: %.6f", best_score)
-            _log.debug("params of the best estimator found: %s", str(best_params))
-            _log.debug("training the model with complete data")
+            if _log.isEnabledFor(logging.DEBUG):
+                _log.debug("score of the best estimator found: %.6f", best_score)
+                _log.debug("params of the best estimator found: %s", str(best_params))
+                _log.debug("training the model with complete data")
+            else:
+                _log.debug("finished hyperopt at %.6f, training the full model", -best_score)
```

Comment:
but... `Logger.debug()` method itself is
```python
        if self.isEnabledFor(DEBUG):
            self._log(DEBUG, msg, args, **kwargs)
```
So this line has no affect. So may be you want to up it to info level?
```suggestion
                _log.info("finished hyperopt at %.6f, training the full model", -best_score)
```

Reply:
or just delete it

Reply:
Yes, should be `info`, thanks